pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.openhab.tools</groupId>
   <artifactId>static-code-analysis</artifactId>
-  <version>0.0.3</version>
+  <version>0.0.4-SNAPSHOT</version>
```

Comment:
The 'problem' with having a normal version here is that your local maven folder contains a different 0.0.3 version than mine, which is normal for snapshots but unexpected for final versions. I can ofcourse revert this line if both of you do not completely agree it's not crucial for me, it just felt cleaner.